pages/common/node.md

```diff
@@ -18,3 +18,7 @@
 - Evaluate and print result, useful to see node's dependencies versions:
 
 `node -p "{{process.versions}}"`
+
+- Enable Node's inspector agent and pause execution until a debugger is connected. Source code is fully compiled:
```

Comment:
What does "Source code is fully compiled" mean in this context? Can we explain it a bit more?

Reply:
Node.js parses the functions inside JavaScript files on first access. The option prevents the delayed parsing and ensures that breakpoints can be validated before running the code. I've clarified this in the latest commit.

Reply:
Sure, but is this something somebody needs to know to use the command? Feels like it makes the description unncessarily long without adding much value.

Reply:
It is not required for debugging. I would recommend no lazy and VSCode defaults to using it because breakpoints don't work in source code areas that haven't been seen (parsed) by Node.js. Since this isn't something found in the man page `man node` I thought to add it.

Reply:
Ah okay, now I understand that this is tied to the --no-lazy option. Yeah okay, this makes sense. But let's change the description to include this in a single sentence. Otherwise it reads like something extra and disconnected from the `--no-lazy` option.

Reply:
See latest commit.